src/mono/mono/mini/mini-llvm.c

```diff
@@ -2562,51 +2574,48 @@ emit_cond_system_exception (EmitContext *ctx, MonoBasicBlock *bb, const char *ex
 		return;
 	}
 
-	callee = ctx->module->throw_corlib_exception;
-	if (!callee) {
```

Comment:
Why removing this check?

Reply:
These were caches, but they were not updated so they never worked.

Reply:
Gotcha!